mcannalte.planx-pla.net/etlMapping.yaml

```diff
@@ -0,0 +1,67 @@
+mappings:
+  - name: mcannalte_etl
```

Comment:
The process today for setting this up is copy/paste. There's a non-zero chance I didn't do everything properly, but I _think_ I got it all